pages/instructorAssessmentInstance/instructorAssessmentInstance.js

```diff
@@ -12,7 +12,7 @@ const sql = sqlLoader.loadSqlEquiv(__filename);
 
 const logCsvFilename = (locals) => {
     return sanitizeName.assessmentFilenamePrefix(locals.assessment, locals.assessment_set, locals.course_instance, locals.course)
-        + sanitizeName.sanitizeString(locals.instance_user.uid)
+        + ((locals.assessment.group_work) ? (locals.group_info.name) : (sanitizeName.sanitizeString(locals.instance_user.uid)))
```

Comment:
Should we sanitize the group name as well? At the moment I think it won't matter, but in the future we might allow extra characters and then this would be a good idea?
```suggestion
        + sanitizeName.sanitizeString(locals.assessment.group_work ? locals.group_info.name : locals.instance_user.uid)
```